test/unit/math/mix/mat/fun/acos_test.cpp

```diff
@@ -0,0 +1,97 @@
+#include <stan/math/mix/mat.hpp>
+#include <gtest/gtest.h>
+#include <test/unit/math/prim/mat/vectorize/prim_scalar_unary_test.hpp>
+#include <test/unit/math/rev/mat/vectorize/rev_scalar_unary_test.hpp>
+#include <test/unit/math/fwd/mat/vectorize/fwd_scalar_unary_test.hpp>
+#include <test/unit/math/mix/mat/vectorize/mix_scalar_unary_test.hpp>
+#include <stan/math/prim/mat/fun/acos.hpp>
+#include <test/unit/math/prim/mat/vectorize/vector_builder.hpp>
+
+/**
+ * This is the structure for testing mock function acos (defined in the
+ * testing framework).  See README.txt for more instructions.
```

Comment:
Does anyone still need me to code review this again?
I'm traveling and pretty booked this week.

- Bob

> On Jun 27, 2016, at 8:24 AM, rayleigh <notifications@github.com> wrote:
> 
> In test/unit/math/mix/mat/fun/acos_test.cpp:
> 
> > @@ -0,0 +1,97 @@
> > +#include <stan/math/mix/mat.hpp>
> > +#include <gtest/gtest.h>
> > +#include <test/unit/math/prim/mat/vectorize/prim_scalar_unary_test.hpp>
> > +#include <test/unit/math/rev/mat/vectorize/rev_scalar_unary_test.hpp>
> > +#include <test/unit/math/fwd/mat/vectorize/fwd_scalar_unary_test.hpp>
> > +#include <test/unit/math/mix/mat/vectorize/mix_scalar_unary_test.hpp>
> > +#include <stan/math/prim/mat/fun/acos.hpp>
> > +#include <test/unit/math/prim/mat/vectorize/vector_builder.hpp>
> > +
> > +/**
> > + * This is the structure for testing mock function acos (defined in the
> > + * testing framework).  See README.txt for more instructions.
> 
> Okay. I got a chance to work on it today and made those changes.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
>